tools/buildtools_wrappers/unused_deps.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Wrapper script for running unused_deps
+
+platform=$(uname)
+
+if [ "$platform" == "Darwin" ]; then
+    UNUSED_DEPS_BINARY=$(pwd)/external/unused_deps_osx/file/unused_deps.osx
+elif [ "$platform" == "Linux" ]; then
+    UNUSED_DEPS_BINARY=$(pwd)/external/unused_deps/file/unused_deps
+else
+    echo "unused_deps does not have binary for $platform"
+    exit 1
+fi
+
+cd $BUILD_WORKSPACE_DIRECTORY
+$UNUSED_DEPS_BINARY $*
```

Comment:
After you run this, does your current folder change?

Reply:
@oferb it doesn't, `bazel run` runs script in a separate sub-shell